app/src/main/java/space/taran/arknavigator/ui/adapter/TagsSelectorAdapter.kt

```diff
@@ -91,6 +91,17 @@ class TagsSelectorAdapter(
         this.isCheckable = true
         this.isChecked = false
         this.setTextColor(Color.BLACK)
+        if (tag.contains("Kind:", ignoreCase = true)) {
```

Comment:
Let's declare text constant containing `"Kind:"` value for easier maintenance in the future.

Reply:
Yes sure.